src/components/command-bar/index.tsx

```diff
@@ -157,6 +157,7 @@ const CommandBarProvider = ({ children }: CommandBarProviderProps) => {
 			setCurrentCommand,
 			setCurrentInputValue,
 			toggleIsOpen,
+			instructionsElementId: 'footer-keyboard-instructions',
```

Comment:
💡 we could use [`useId` ](https://react.dev/reference/react/useId) or update this to not mention the footer as the descriptive element may change.

Reply:
Should we called this `labelledById` or something? The name confuses me a bit. I wouldn't know straight away what this is used for unless I looked at the interface directly with the code comment.

Reply:
1. I'll leave this `'footer-keyboard-instructions'` hard-coded value in this PR. We can easily change this post-PR if we need to!
2. I thought about `labelledById` a bit... but I think `instructionsElementId` (the previous variable name) is actually reasonable! In my head, the naming pans out to: _"the footer div is the `instructionsElementId` and the dialog is labelled by `instructionsElementId`"_